frontend/src/components/ui/ProductCreateDialog.tsx

```diff
@@ -0,0 +1,113 @@
+import { Dialog, DialogTitle, DialogContent, Box, Typography, TextField, MenuItem, DialogActions, Button, Grid, Avatar } from "@mui/material";
+import { useState } from "react";
+
+interface CreateRequestDialogProps {
+    open: boolean;
+    onClose: () => void;
+    onSubmit: (data: any) => void;
+  }
+  
+  const fakeCategories = ['Ноутбуки', 'Телефоны', 'Видеокарты', 'Роботы'];
+  
+  const CreateRequestDialog = ({ open, onClose, onSubmit }: CreateRequestDialogProps) => {
+    const [form, setForm] = useState({
+      images: [] as File[],
+      title: '',
+      category: '',
+      description: '',
+      address: '',
+      price: '',
+    });
```

Comment:
Тут тоже можно объявить тип данных всей формы и не использовать `any`
```suggestion
interface FormInputs {
  images: File[];
  title: string;
  category: string; // а лучше enum (даже готовый в моделях есть)
  description: string;
  address: string;
  price: string;
}

interface CreateRequestDialogProps {
    open: boolean;
    onClose: () => void;
    onSubmit: (data: FormInputs) => void;
  }
  
  const fakeCategories = ['Ноутбуки', 'Телефоны', 'Видеокарты', 'Роботы'];
  
  const CreateRequestDialog = ({ open, onClose, onSubmit }: CreateRequestDialogProps) => {
    const [form, setForm] = useState<FormInputs >({
      images: [],
      title: '',
      category: '',
      description: '',
      address: '',
      price: '',
    });
```